src/common/transformations/src/transformations/common_optimizations/moc_transformations.cpp

```diff
@@ -155,6 +156,7 @@ bool ov::pass::MOCTransformations::run_on_model(const std::shared_ptr<ov::Model>
     REGISTER_PASS(manager, ConvertNmsGatherPathToUnsigned)
     REGISTER_PASS(manager, StridedSliceOptimization, m_use_shapes)
     REGISTER_PASS(manager, BroadcastElementwiseFusion)
+    REGISTER_PASS(manager, BroadcastTransition)
```

Comment:
do we know the reason why it wasn't enabled in other plugins? e.g gna?

Reply:
maybe @v-Golubev can comment on that

Reply:
or @mryzhov ?

Reply:
Because it was not required in our networks

Reply:
Discussed with Mikhail Ryzhov, in case of any issues with e2e tests, we can disable this transformation for specific plugins

Reply:
If I am not mistaken, this transformation was merged into master at the end of release cycle and we decided not to risk potential regressions in the plugins for which we could not validate changes locally.